int64 id
string name_r
string alias_r
#Nombre del move_group al que pertenece
string name_mg
int64 DOF_mg
ltlplanner/limit[] limits
int64 idregionparent
int64[] idregionchildren
int64[] idneighbours
